frontend/test/jest/tests/ProductListItem.test.tsx

```diff
@@ -0,0 +1,38 @@
+// @ts-nocheck
+import { render, screen } from '@testing-library/react';
+import { ProductListItem } from '@components/product-list/sections/FilterableProductsSection/ProductList';
+import * as ProductSearch from '@components/product-list/sections/FilterableProductsSection/useProductSearchHitPricing';
+import * as AppContext from '@ifixit/app';
+import { mockProduct } from 'test/jest/__mocks__/mockProduct';
+
+describe('renders ProductListItem', () => {
+   beforeEach(() => {
+      ProductSearch.useProductSearchHitPricing = jest.fn().mockReturnValue({
+         price: 5,
+         compareAtPrice: 5,
+         isDiscounted: 5,
+         percentage: 5,
+      });
+
+      AppContext.useAppContext = jest
+         .fn()
+         .mockReturnValue({ ifixitOrigin: 'https://www.cominor.com' });
+   });
+
+   afterEach(() => {
+      jest.clearAllMocks();
+   });
+
+   /**
+    * @jest-environment jsdom
```

Comment:
Why do we need this?

Reply:
It's from here https://jestjs.io/docs/25.x/configuration#testenvironment-string but looks like we don't need it anymore because the default is `jsdom`. Removed it in https://github.com/iFixit/react-commerce/pull/438/commits/8c37f35950332ac60c52cd675fbd297ff026d78d.